templates/test/ci/cluster-template-prow-machine-pool-ci-version.yaml

```
@@ -123,7 +116,7 @@ spec:
           echo "* testing CI version $$CI_VERSION"
           # Check for semver
           if [[ "$${CI_VERSION}" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
-            CI_URL="gs://kubernetes-release/release/$$CI_VERSION/bin/linux/amd64"
+            CI_URL="https://storage.googleapis.com/kubernetes-release/release/$${KUBERNETES_VERSION}/bin/linux/amd64"
```

```suggestion
            CI_URL="https://storage.googleapis.com/kubernetes-release/release/$${CI_VERSION}/bin/linux/amd64"
```

We're not defining `KUBERNETES_VERSION` anywhere, our variable is called `CI_VERSION`, this is what is causing the test to fail with `/tmp/kubeadm-bootstrap.sh: line 38: KUBERNETES_VERSION: unbound variable`

same thing applies to the other files